server/models/db/add-worker-timeslot.js

```diff
@@ -0,0 +1,18 @@
+const _ = require('lodash');
+
+const { db } = require('../../lib/connection');
+
+const Tables = require('../../constants/tables.json');
+const AddTimeslotStatus = require('../../constants/timeslot-status.json');
```

Comment:
I would rename this to TimeslotStatus to avoid confusion as it contains the possible values of the timeslot status rather than performs a manipulation.

Reply:
Appreciate the level of detail. Very helpful!